kratos/python/add_utilities_to_python.cpp

```diff
@@ -57,6 +57,18 @@ namespace Kratos
 namespace Python
 {
 
+/**
+ * @brief Sets the current table utility on the process info
+ * @param rCurrentProcessInfo The process info
+ */
+void SetOnProcessInfo(
```

Comment:
why do you need this function?
Isn't it easier to do it directly?

Reply:
Yes, but it can be done right now due to #2033